.circleci/config.yml

```diff
@@ -59,8 +59,12 @@ jobs:
           command: |
             . venv/bin/activate
             cd python
-            python3 -m pytest ml4ir/applications/classification/tests --junitxml=test_results/results.xml
+            COVERAGE_FILE=reports/.coverage.classif coverage run  --include "ml4ir/*" -m pytest ml4ir/applications/classification/tests/ --junitxml=test_results/results.xml
```

Comment:
We use an env variable `COVERAGE_FILE` to control the name of the file that results are written.